### DISCUSSION TITLE
Examples of Nova websites with significant layout changes

### DISCUSSION THREAD

**Discussion by mariusghitoiu** [laravel/nova-issues#3487] [General]
Hi, 

We have a project with Laravel Nova where we have focused more on the backend flows. We have imports, exports, pdf generation, custom filters, updated CRUD operations and all sort of actions to do stuff.

We have done all these without changing almost anything on the visual side of things. We have the default theme. 

Now, we are getting to a point where we want to update the flows to be a bit more intuitive for the user and I am lacking inspiration on how much Nova can be customized. 

I have opened this tread looking for inspiring examples, screenshots not necessary code, from other Nova users on the visual part.

**Comment by KasparRosin** [comment]
@mariusghitoiu 👋🏻 

![Screenshot 2021-08-06 at 16 21 20](https://user-images.githubusercontent.com/33309407/128516639-7b889741-1880-4c9b-b6f9-95dbede3f06a.png)

Not sure if this is what you were looking for, but here is an example of how we @optimistdigital made nova resources more compact for internal nova projects. 

I don't recommend making large changes if you plan to keep up to date with latest nova releases.
Currently while they are working on next large version of nova the releases have gotten more rare, but they used to make releases weekly, sometimes even multiple times a week and the more components you change, the more manual work it will require to fix merge conflicts.

**Reply by mariusghitoiu** [reply]
Thank you @KasparRosin. 
On this layout, do you still have the menu bar on the left? 

True, I have mixed feelings about changing the layout now, knowing that a new major version of Nova will be released. 
Not sure when, though, hopefully this year.
